modern/org/commcare/modern/database/TableBuilder.java

```diff
@@ -187,9 +187,9 @@ public static String scrubName(String input) {
         ByteArrayOutputStream bos = new ByteArrayOutputStream();
         try {
             externalizable.writeExternal(new DataOutputStream(bos));
-        } catch (IOException e1) {
-            e1.printStackTrace();
-            throw new RuntimeException("Failed to serialize externalizable for content values");
+        } catch (IOException e) {
+            throw new RuntimeException("Failed to serialize externalizable " + externalizable +
```

Comment:
Hey,

One thing that helps a _lot_ in cases like this is to not just throw the message, but to include the full trace. The best way to do so is to create the runtime exception, and set its "cause" to the original throwable, which comes with a whole host of benefits. This should definitely be a best-practice when we deal with this kind of error handling.